### ISSUE TITLE
Non reporting plugins are not added to runtime options

### ISSUE THREAD

**Issue by andrcuns** [prashant-ramcharan/courgette-jvm#40]
It seems that all non reporting plugins are not added to final runtime options.
Looks like the problem is in this if statement which only executes and adds plugins when plugin starts with either `html:` or `json:`:
```java
asList(plugins).forEach(plugin -> {
            if (isReportPlugin.test(plugin)) {
                if (cucumberFeature != null) {
                    pluginList.add(plugin);

                    String extension = plugin.substring(0, plugin.indexOf(":"));

                    if (!extension.equals("")) {
                        final String reportPath = String.format("%s:%s.%s", extension, getMultiThreadReportFile(), extension);
                        pluginList.add(reportPath);
                    }
                } else {
                    pluginList.add(plugin);
                }
            }
        });
```

When debugging, I also see that command doesn't even contain `pretty` plugin even though it is provided in options.

**Comment by prashant-ramcharan** [comment]
Yes, the else statement needs to go one block down -- good spot!

**Comment by prashant-ramcharan** [comment]
This is now fixed in 2.1.0
